src/js/weather.js

```diff
@@ -8,16 +8,36 @@ const city = document.querySelector('.city');
 export async function getWeather(city) {  
     weatherIcon.className = 'weather-icon owf';
     const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&lang=ru&appid=f793b145b04b72ffdfd775747be4fac1&units=metric`;
-    const res = await fetch(url);
-    const data = await res.json(); 
-
-    weatherIcon.classList.add(`owf-${data.weather[0].id}`);
-    temperature.textContent = `${Math.round(data.main.temp)}`;
-    weatherDescription.textContent = `${data.weather[0].description}`;
-    wind.textContent = `скорость ветра: ${data.wind.speed} м/c`;
-    humidity.textContent = `влажность: ${data.main.humidity} %`;
+    try {
```

Comment:
It is good, that you decided to improve this part, but I recommend to keep the scope of the PR consistent and not add new staff until you resolve all the existing things. new staff should go to a separate PR. no need to remove it from here, just for the future